sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/policy/VersionPolicy.java

```diff
@@ -0,0 +1,28 @@
+package com.azure.storage.common.implementation.policy;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpPipelinePosition;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import reactor.core.publisher.Mono;
+
+public class VersionPolicy implements HttpPipelinePolicy {
```

Comment:
Eventually roll this back when generator adds support for it

Reply:
Mind filing an issue against Java's code generator for each location, or set of common locations, where a transform can't be applied to produce the equivalent API as the previous generator allowed. This will make sure we have the features necessary as we begin porting more clients to the new generator.

CC @srnagar @jianghaolu

Reply:
Yep, I already have an issue open https://github.com/Azure/autorest.java/issues/889

Reply:
@alzimmermsft  - there's already an issue for this - https://github.com/Azure/autorest.java/issues/889